packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -639,6 +646,7 @@ export class RemotedFluidDataStoreContext extends FluidDataStoreContext {
                     throw new Error(`Invalid snapshot format version ${snapshotFormatVersion}`);
                 }
                 this.pkg = pkgFromSnapshot;
+                this._isRootDataStore = isRootDataStore ?? false;
```

Comment:
I think !!isRootDataStore  is more readable

Reply:
As per Tony's comment, I am changing the default value to true if this attribute doesn't exists. This will make sure that root data stores in older documents are not garbage collected incorrectly.